addons/mail/static/src/discuss/core/web/user_menu_patch.scss

```diff
@@ -0,0 +1,8 @@
+.o-web-UserMenu-imStatus {
+    bottom: -2px;
+    right: -2px;
+}
+
+.o-away {
+    color: $yellow;
```

Comment:
This is strange, why do we only have to define one color? Why is it not text-danger there?

Reply:
The away is the yellow/orange color, whereas `text-danger` is red.
The closest is `warning` but its color is brown in dark theme.

I really think the color should be fixed for IM status, there's no point to couple them with theme colors.

Reply:
Ok, but `o-away` is a very generic selector though. Either it should be i na more generic css file, or it should be made specific to this component.